core/src/Cabana_LinkedCellList.hpp

```diff
@@ -75,21 +76,33 @@ class LinkedCellList
       \param grid_min Grid minimum value in each direction.
 
       \param grid_max Grid maximum value in each direction.
+
+      \param neighborhood_radius Radius for neighbors
+
+      \param cell_size_ratio Ratio of the cell size to the neighborhood size
     */
     template <class SliceType>
     LinkedCellList(
         SliceType positions, const typename SliceType::value_type grid_delta[3],
         const typename SliceType::value_type grid_min[3],
         const typename SliceType::value_type grid_max[3],
+        const Scalar neighborhood_radius, const Scalar cell_size_ratio = 1,
```

Comment:
Let's default the cutoff to be equal to the `grid_delta`, first because it's most often how we use it and second because it keeps backwards compatibility for free

Reply:
Do you mean default the `neighborhood_radius` to be the `grid_delta` similar to defaulting the `cell_size_ratio` to 1? Or do you mean assume that they are equal implicitly and when the `neighborhood_radius` is used in the constructor to replace it with the `grid_delta`? I don't believe the first option is legal since  `grid_delta` is an input to this function

Reply:
You're right - we can just add another constructor without the cutoff argument that defaults it during construction